index.html

```diff
@@ -0,0 +1,157 @@
+                people (2004).[6] Geographically embedded on the eastern foothills of the Anti-Lebanon mountain range 80
+                kilometres (50 mi) inland from the eastern shore of the Mediterranean on a plateau 680 metres (2,230 ft)
+                above sea level, Damascus experiences a semi-arid climate because of the rain shadow effect. The Barada
+                River flows through Damascus.
+                First settled in the second millennium BC, it was chosen as the capital of the Umayyad Caliphate from
+                661 to 750. After the victory of the Abbasid dynasty, the seat of Islamic power was moved to Baghdad.
+                Damascus saw a political decline throughout the Abbasid era, only to regain significant importance in
+                the Ayyubid and Mamluk periods. Today, it is the seat of the central government and all of the
+                government ministries. As of 2018, Damascus has witnessed repeated conflicts and has been considered by
+                Mercer as one of the most unfavorable places to live.[7]
+
+
+            </p>
+        </section>
+    </section>
+    <section class="sec2">
+        <section class="sectext">
+            <a name="history"></a>
+            <h2>Damscus</h2>
+            <p>
+                The name of Damascus first appeared in the geographical list of Thutmose III as
+                T-m-ś-q in the 15th century BC.[8] The etymology of the ancient name "T-m-ś-q" is uncertain. It is
+                attested as 𒀲𒋙 Imerišú in Akkadian, T-m-ś-q in Egyptian, Dammaśq (דמשק) in Old Aramaic and
+                Dammeśeq (דמשק) in Biblical Hebrew. A number of Akkadian spellings are found in the Amarna letters, from
+                the 14th century BC: Dimasqa, Dimàsqì, and Dimàsqa. Later Aramaic spellings of
+                the name often include an intrusive resh (letter r), perhaps influenced by the root dr, meaning
+                "dwelling". Thus, the English and Latin name of the city is "Damascus" which was imported from (Greek:
+                Δαμασκός) originated from "the Qumranic Darmeśeq (דרמשק), and Darmsûq in Syriac[9][10]",
+                meaning "a well-watered land".[11] In Arabic, the city is called Dimašqu š-Šāmi (دمشق الشام), although
+                this is often shortened to either Dimašq or aš-Šām by the citizens of Damascus, of Syria and other Arab
+                neighbors and Turkey (as Şam). Aš-Šām is an Arabic term for "Levant" and for "Syria"; the latter, and
+                particularly the historical region of Syria, is called Bilādu š-Šāmi (بلاد الشام / "land of the
+                Levant"). Historically, Baalshamin or Ba'al Šamem (Aramaic:translit. Lord of
+                Heaven(s)),[12][13] was a Semitic sky-god in Canaan/Phoenicia and ancient Palmyra.[14][15] Hence, Sham
+                refers to (heaven or sky).
+
+
+
+
+            </p>
+        </section>
+    </section>
+    <section class="sec3">
```

Comment:
avoid numbers in class names. either name all of them sec instead of sec1 sec2 sec3 or use and id